modules/porous_flow/doc/content/source/bcs/PorousFlowEnthalpySink.md

```diff
@@ -0,0 +1,28 @@
+# PorousFlowEnthalpySink
+
+`PorousFlowEnthalpySink` implements a sink that adds heat energy corresponding to
+adding fluid at a mass flux rate (computed by a function) at a specified temperature.
+
+This object should be used in conjunction with (PorousFlowSink)[PorousFlowSink.md]
+that uses the same `flux_function`, so that the correct amount of fluid is injected into the system.
+
+Note that the fluid property object used by this boundary condition should be the same one that is
+used in the computational domain where this object is located.
+
+Parameter [!param](/BCs/PorousFlowEnthalpySink/pressure)
+(along with [!param](/BCs/PorousFlowEnthalpySink/T_in)) is used to calculate
+the injected fluid enthalpy. Most frequently, it is the PorousFlow pressure variable (the porepressure
+in the porous medium). This models the situation where fluid is injected at a specified rate and
+temperature (using this BCs and a PorousFlowSink) which potentially
+leads to changes in the porepressure. Alternately, it may be the pressure of the injected fluid, as
+fixed by an external agent (such as a pump) which is stored in an AuxVariable.
```

Comment:
Something's gone amiss here, i think (i might be getting confused with versions).  The code appears to model only the first situation, while the AuxVariable `_pressure` is not actually used.

Reply:
We built this for case 1. But, I think you were suggesting that case 2 can also be modeled back on July. But we never tried, so maybe just strip this bit?

Reply:
Also, I think, because we needed the derivative of pressure, we switched into use `_pp` and `_phase` and therefore "pressure" parameter is no longer needed. Can you confirm this?

Reply:
I think we still need to capture both cases.  Perhaps the user can supply either `pressure` xor `phase`.  If the latter then the code is as written.  If the former then use `pressure` instead of your `_pp`, and the derivatives will be zero.

Reply:
Ok. Sounds good. I will add the case (2).